ncpi_fhir_utility/update_versions.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""
+Updates versions and dates in all of our FHIR resources.
+
+Example:
+    $ scripts/update_versions.py 1.1.1
+"""
+import json
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+
+import click
+
+
+def bump_ig_version(filePath, new_version):
```

Comment:
I think you can use Python's ConfigParser to read `.ini` files? https://docs.python.org/3.7/library/configparser.html

Reply:
√